src/core/layout/scrollable-list/scrollable-list-handlers.js

```diff
@@ -10,27 +10,19 @@ import fp from "../../../../lib/lodash/fp/fp.js";
 
 const WHEEL_SCROLL_FACTOR = 0.005;
 
-const handleClickIfVisible = (gelButton, scene, handler) => () => {
-    if (!gelButton.rexContainer.parent) return;
+const updatePanelOnScroll = panel => () =>
+    getPanelItems(panel).map(item => enableOrDisableButton(panel, item).setElementSizeAndPosition());
 
-    if (isA11yClick(scene)) {
-        handler(gelButton);
-        return;
-    }
-
-    const panel = gelButton.rexContainer.parent.getTopmostSizer();
-    const safeArea = scene.layout.getSafeArea({}, false);
-    const height = scene.scale.displaySize.height;
-    const topY = height / 2 + safeArea.y + panel.space.top;
-    const bottomY = topY + panel.innerHeight;
-    const mouseY = scene.input.y;
-    if (mouseY >= topY && mouseY <= bottomY) handler(gelButton);
+const isItemVisibleInPanel = (panel, item) => {
+    const visibleBounds = getVisibleRangeBounds(panel);
+    const itemBounds = getItemBounds(panel, item);
+    return !(
+        itemBounds.lower + item.height < visibleBounds.lower || itemBounds.upper - item.height > visibleBounds.upper
+    );
 };
 
-const isA11yClick = scene =>
-    scene.input.activePointer.id === 0 || scene.sys.time.now - scene.input.activePointer.upTime > 50;
-
-const updatePanelOnScroll = panel => () => getPanelItems(panel).map(item => item.setElementSizeAndPosition());
+const enableOrDisableButton = (panel, item) =>
```

Comment:
OK this makes lots of sense- we're turning off the buttons when they're scrolled off the top / bottom.